execution.bs

```diff
@@ -4377,46 +4399,25 @@ enum class forward_progress_guarantee {
 
 ### `execution::set_stopped` <b>[exec.set_stopped]</b> ### {#spec-execution.receivers.set_stopped}
 
-1. `execution::set_stopped` is used to send a <i>stopped signal</i> to a receiver.
+1. `execution::set_stopped` is used to send a stopped completion signal to a receiver.
 
 2. The name `execution::set_stopped` denotes a customization point object. The expression `execution::set_stopped(R)` for some subexpression `R` is expression-equivalent to:
 
-    1. `tag_invoke(execution::set_stopped, R)`, if that expression is valid. If the function selected by `tag_invoke` does not signal the receiver `R`’s stopped channel, the behavior of calling `execution::set_stopped(R)` is undefined.
+    1. `tag_invoke(execution::set_stopped, R)`, if that expression is valid. The function selected by `tag_invoke` shall signal the receiver `R`’s stopped channel.
 
         * <i>Mandates:</i> The `tag_invoke` expression above is not potentially
             throwing.
 
     2. Otherwise, `execution::set_stopped(R)` is ill-formed.
 
-### Receiver queries <b>[exec.recv_queries]</b> ### {#spec-execution.receivers.queries}
-
-#### `execution::forwarding_receiver_query` <b>[exec.recv_queries.forwarding_receiver_query]</b> #### {#spec-execution.receivers.queries.forwarding_receiver_query}
-
-1. `execution::forwarding_receiver_query` is used to ask a customization point object whether it is a receiver query that should be forwarded through receiver adaptors.
-
-2. The name `execution::forwarding_receiver_query` denotes a customization point object. For some subexpression `t`, `execution::forwarding_receiver_query(t)` is expression equivalent to:
-
-    1. `tag_invoke(execution::forwarding_receiver_query, t)`, contextually converted to `bool`, if the `tag_invoke` expression is well formed.
-
-        * <i>Mandates:</i> The `tag_invoke` expression is indeed contextually
-            convertible to `bool`, that expression and the contextual conversion
-            are not potentially-throwing and are core constant expressions if
-            `t` is a core constant expression.
-
-    2. Otherwise, `false` if the type of `t` is one of `set_value_t`, `set_error_t`, or `set_stopped_t`.
-
-    3. Otherwise, `true`.
-
-3. [*Note:* Currently the only standard receiver query is `execution::get_env` -- *end note*]
-
 ## Operation states <b>[exec.op_state]</b> ## {#spec-execution.op_state}
 
 1. The `operation_state` concept defines the requirements for an operation state type, which allows for starting the execution of work.
 
     <pre highlight=c++>
     template&lt;class O>
       concept operation_state =
-        destructible&lt;O> &&
+        queryable&lt;O> &&
```

Comment:
Why `sender` as `get_attrs` but `operation_state` and `scheduler` are directly querable. It would be good to explain it in design. From what I understand:
* `scheduler` is already cheap handle, so it could be it own querable
* `operation_state` is persistent until operation finishes, so we want to tie lifetime of the properties to it.